_posts/zh/newsletters/2018-07-31-newsletter.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Bitcoin Optech Newsletter #6'
+permalink: /zh/newsletters/2018/07/31/
+name: 2018-07-31-newsletter-zh
+slug: 2018-07-31-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的新闻简报包括常规的仪表板和行动项目，开发人员 Anthony Towns 关于 Xapo 合并 400 万个 UTXO 的特写文章，有关比特币脚本系统可能升级的新闻，一些在比特币堆栈交换上高票问题和回答的链接，以及 Bitcoin Core、Lightning Network Daemon (LND) 和 C-lightning 项目的开发分支中的一些值得注意的提交。
+
+## 行动项目
+
+- **<!--bitcoin-core-0.16.2-released-->发布 Bitcoin Core 0.16.2**：一个小版本，带来了错误修复和小改进。如果您使用 [abandontransaction][rpc abandontransaction] 或 [verifytxoutproof][rpc verifytxoutproof] RPC，您应该特别考虑升级。否则，我们建议您查看[发布说明][bitcoin core 0.16.2]了解可能影响您操作的其他变化，并在方便时升级。
+
+## 仪表板项目
+
+- **<!--fees-still-low-->费用仍然低**：在周日结束的 2016 区块重新目标周期中，哈希率增加了超过14%，使区块间平均时间为 8 分 41 秒。这有助于吸收过去一周价格投机带来的交易量增加。紧随难度重新目标之后，区块间平均时间恢复为 10 分钟。
+
+  随着我们从正常的周末交易低迷过渡到新的一周，估计的交易费用可能会迅速增加。我们建议在靠近周末时再发送大量低费用交易（如合并），那时交易量开始再次下降。
+
+## 实地报告：Xapo 合并 400 万个 UTXO
+
+*由 [Xapo][] 的 Bitcoin Core 开发人员 [Anthony Towns](https://twitter.com/ajtowns) 编写*
+
+{% include articles/towns-xapo-consolidation.md hlevel='###' %}
```

Comment:
@bittomhan I think it makes sense to also translate this field report from AJ into Chinese as well.

Here is a Chinese translation of the field report in newsletter 277 which could be helpful to reference:
https://github.com/bitcoinops/bitcoinops.github.io/pull/1408/files

What do you think?

Reply:
I'm glad to have been reminded, as I previously hadn't realized that the content of these articles also needed to be translated into Chinese. I have now translated and submitted a PR. After the PR for the xapo article is approved, I will add the corresponding file index path to it. Thank you very much!

Reply:
This needs a reference to the zh version instead